test/test_transforms.py

```diff
@@ -1959,10 +1959,12 @@ def test_autoaugment(self):
     def test_random_erasing(self):
         img = torch.ones(3, 128, 128)
 
-        t = transforms.RandomErasing(scale=(0.1, 0.1), ratio=(1. / 3., 3. / 1.))
+        t = transforms.RandomErasing(scale=(0.1, 0.1), ratio=(1. / 3., 3.))
         y, x, h, w, v = t.get_params(img, t.scale, t.ratio, [t.value, ])
         aspect_ratio = h / w
-        self.assertTrue(aspect_ratio > 1. / 3. and aspect_ratio < 3. / 1.)
+        # Add some tolerance due to the rounding and int conversion used in the transform
+        tol = 0.05
+        self.assertTrue(1. / 3. - tol <= aspect_ratio <= 3. + tol, aspect_ratio)
```

Comment:
Can we lower the tolerance (I haven't looked at the details)?

```suggestion
        tol = 0.05
        self.assertTrue(1. / 3. - tol <= aspect_ratio <= 3. + tol)
```

Also just curious, any reason to use e.g. `1. / 3.) when it's equivalent to `1 / 3`?

Reply:
I tried but this is the lowest the tolerance can go. The issue is caused by the following roundings:
https://github.com/pytorch/vision/blob/06a5858b3b73d62351456886f0a9f725fddbb3fe/torchvision/transforms/transforms.py#L1586-L1587

The formatting of `1. / 3.` was used by the original author of the test and I did not change it. I have now reformatted it.

Good catch on the `aspect_ratio`, it was left from the investigation. ✌️